templates/generic_show_form.html.twig

```diff
@@ -789,6 +789,17 @@
                      ) }}
                   {% endif %}
 
+                  {% if item.isField('stock_target') %}
```

Comment:
i think this addition should be in a specific twig template like done in `templates/pages/assets/monitor.html.twig`